spec/context/dimension.fmf

```diff
@@ -0,0 +1,50 @@
+summary: Supported context dimensions
+
+story:
+    As a tester I want to clearly specify dimensions, such as
+    'distro' or 'trigger', so that I can well describe the
+    context in which a test is running.
+
+description: |
+    The following dimensions are supported:
```

Comment:
What means `supported`? Just that the name is reserved and has a  canonical meaning or does it also implies that TMT sets (read: is going to set ) the dimension value?

Reply:
I've updated the introduction paragraph to make it more clear.